cli-plugins/apps/reference/clustersupplychain.hbs.md

```diff
@@ -2,16 +2,17 @@
 
 This command provides details about the cluster supply chain.
 
-## Tanzu apps cluster supply chain list
+## tanzu apps clustersupplychain list
 
 The `tanzu apps clustersupplychain list` command lists the available supply chains installed in the
-cluster (supported clustersupplychain alias is `csc`).
+cluster (supported clustersupplychain aliases are `csc`, `cluster-supply-chain`, 
+`cluster-supply-chains`, `clustersupplychains`).
 
 Run the following command to view more detailed information about the selectors and conditions that
 must be met for a workload to be selected by a certain supply chain:
 
 ```console
-tanzu apps clustersupplychain get SUPPLYCHAIN-NAME`.
+tanzu apps clustersupplychain get SUPPLYCHAIN-NAME
```

Comment:
always use the command name that's listed by default in help rather than an alias.

```console
tanzu apps cluster-supply-chain get SUPPLYCHAIN-NAME
```

Reply:
pk, used the default in help